Sources/SwiftGRPCNIO/GRPCServer.swift

```diff
@@ -12,7 +12,9 @@ public final class GRPCServer {
     hostname: String,
     port: Int,
     eventLoopGroup: EventLoopGroup,
-    serviceProviders: [CallHandlerProvider]) -> EventLoopFuture<GRPCServer> {
+    serviceProviders: [CallHandlerProvider],
+    errorDelegate: ServerErrorDelegate? = nil
```

Comment:
How about a simple "LoggingErrorDelegate" (that just logs errors) instead of `nil` as the default?